x-pack/plugins/infra/public/alerting/inventory/index.ts

```diff
@@ -15,15 +15,15 @@ import {
 
 import { ObservabilityRuleTypeModel } from '../../../../observability/public';
 
-import { AlertTypeParams } from '../../../../alerting/common';
+import { AlertTypeParams as RuleTypeParams } from '../../../../alerting/common';
```

Comment:
And this is just until things change on the alerting side?

Reply:
Ys exactly. We also talked with them, they will use these names too, so we (or they) can just remove " .... as" part once they did the refactoring on their side.